Geometry/TrackerNumberingBuilder/plugins/CmsTrackerLevelBuilder.h

```diff
@@ -28,13 +27,16 @@ class CmsTrackerLevelBuilderHelper {
  * Abstract Class to construct a Level in the hierarchy
  */
 template <class FilteredView>
-class CmsTrackerLevelBuilder : public CmsTrackerAbstractConstruction<FilteredView> {
+class CmsTrackerLevelBuilder {
 public:
-  void build(FilteredView&, GeometricDet*, const std::string&) override;
-  ~CmsTrackerLevelBuilder() override {}
+  virtual void build(FilteredView&, GeometricDet*, const std::string&);
+  virtual ~CmsTrackerLevelBuilder() = default;
 
 private:
+  static bool skipFirstChild;
```

Comment:
Non-const static variable would be thread hostile.

Reply:
Hi @makortel, It certainly is. That was one of the concerns. @ianna can this task with the current workflow be executed in multiple threads?